src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1434,19 +1438,36 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
         ///     This property forces all AST nodes that generated from this node
         ///     to be executed, even there is no change in AST nodes.
         /// </summary>
-        public virtual bool ForceReExecuteOfNode
+        [Flags]
+        public enum ExecutionHints
+        {
+            None = 0,
+            GenerateAst = 1,
+            ForceExecute = 3
```

Comment:
As you use bit operation, ``ForceExecute`` flag actually include ``GenerateAst`` ?